app/containers/Sessions/containers/SettingsSessionPage/components/SessionSettings.js

```diff
@@ -222,40 +222,48 @@ const SessionSettings = ({
 
               {type === SessionTypes.SMS_SESSION && (
                 <InputContainer mt={15}>
-                  <Row justify="between" align="center" mb={-20}>
-                    <H3 fontWeight="regular">
-                      {formatMessage(messages.smsCode)}
-                    </H3>
-                  </Row>
-                  {smsCodesAttributes.map(({ smsCode }, index) => (
-                    <>
-                      <Row justify="end" align="center">
-                        <TextButton
-                          fontWeight="bold"
-                          hoverDecoration="underline"
-                          clickable
-                          buttonProps={{
-                            color: themeColors.secondary,
-                          }}
-                          onClick={() =>
-                            updateSmsCode(getRandomString(7), index)
-                          }
-                        >
-                          Randomize code
-                        </TextButton>
-                      </Row>
-                      <Row>
-                        <Input
-                          disabled={!editingPossible}
-                          width="100%"
-                          placeholder={formatMessage(messages.smsCode)}
-                          value={smsCode}
-                          onBlur={(val) => updateSmsCode(val, index)}
-                          px={12}
-                        />
-                      </Row>
-                    </>
-                  ))}
+                  {smsCodesAttributes.map(
+                    ({ smsCode, id, clinic: { name: clinicName } }, index) => {
+                      const properMessage = clinicName
+                        ? messages.smsCodeForClinic
+                        : messages.smsCode;
+                      return (
+                        <>
+                          <Row justify="between" align="center" key={id}>
```

Comment:
Because the items that map creates consist of two sibbling Row component, they should be wrapped in a parent component and there should the `key` attribute be added
```suggestion
                        <div key={id}>
                          <Row justify="between" align="center">
```